diff --git a/bootanimation/generate-bootanimation.sh b/bootanimation/generate-bootanimation.sh
index 4a4a742..356de2d 100755
--- a/bootanimation/generate-bootanimation.sh
+++ b/bootanimation/generate-bootanimation.sh
@@ -4,7 +4,7 @@ WIDTH="$1"
 HEIGHT="$2"
 HALF_RES="$3"
 WORKING_DIR="$4"
-OUT="$WORKING_DIR/obj/BOOTANIMATION"
+OUT="$ANDROID_PRODUCT_OUT/obj/BOOTANIMATION"
 
 if [ "$HEIGHT" -lt "$WIDTH" ]; then
     IMAGEWIDTH="$HEIGHT"
